src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -333,56 +333,35 @@ export class CalciteRadioButton {
   //
   // --------------------------------------------------------------------------
 
-  private renderLabel(): VNode {
-    if (this.el.textContent) {
-      return (
-        <calcite-label
-          dir={getElementDir(this.el)}
-          disable-spacing
-          disabled={this.disabled}
-          for={`${this.guid}-input`}
-          layout="inline"
-          scale={this.scale}
-        >
-          <slot />
-        </calcite-label>
-      );
-    }
-    return <slot />;
-  }
-
   render(): VNode {
     const value = this.value?.toString();
 
     return (
       <Host>
-        <div class={{ container: true, "container--labeled": !!this.el.textContent }}>
-          <input
-            aria-label={value || this.guid}
-            checked={this.checked}
-            disabled={this.disabled}
-            hidden={this.hidden}
-            id={`${this.guid}-input`}
-            name={this.name}
-            onBlur={this.onInputBlur}
-            onFocus={this.onInputFocus}
-            ref={this.setInputEl}
-            required={this.required}
-            type="radio"
-            value={value}
-          />
-          <calcite-radio
-            checked={this.checked}
-            disabled={this.disabled}
-            focused={this.focused}
-            hidden={this.hidden}
-            hovered={this.hovered}
-            ref={(el) => (this.radio = el)}
-            scale={this.scale}
-            theme={this.theme}
-          />
-          {this.renderLabel()}
-        </div>
+        <input
+          aria-label={value || this.guid}
```

Comment:
Do we need an aria-label here? Shouldn't the radio button be wrapped in a label or assigned an aria-label itself?

Reply:
If we do need a label, would a separate prop be best? I'm not sure 'value' would always make sense. If the value is some ID, it's not very screen reader friendly.

Reply:
Good question, I'm wondering if I can just do `this.el.getAttribute("aria-label")` and not even define a specific prop here, but like you said the user should provide the label either as a wrapper or with the for attribute.

Reply:
I'm thinking since this input is internal, it shouldn't need this attribute because the label would need to be on the radio-button element instead.  I'll remove it and see if accessibility tests still pass.